example/test/js_routes_example_web/integration/phoenix_jsroutes_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule JsRoutesExampleWeb.PhoenixJsRoutesTest do
+  use JsRoutesExampleWeb.IntegrationCase
+
+  test "have access to router functions", %{session: session} do
+    routes =
+      session
+      |> visit("/")
+      |> get_visible_routes()
+
+    assert length(routes) == 3
+    assert routes == default_routes()
+  end
+
+  test "recompiles the js router when the router changes", %{session: session} do
+    routes =
+      session
+      |> visit("/")
+      |> get_visible_routes()
+
+    assert length(routes) == 3
+
+    new_routes = "
+    get \"user/:id\", TestUserController, :index
+    post \"user/:userId/projects/:id\", TestUserProjectsController, :update
```

Comment:
both lines here need a forward slash at the beginning. i.e. `get  \"/user/:id\", TestUserController, :index`